### PR TITLE
[webui] base.scss: cleanup

### PR DESCRIPTION
**Pull request by avindra**
* Drop vendor prefixes for box-shadow and border-radius
 * Use nesting feature of sass for clarity
 * Drop empty rulesets
 * Group related selectors together that were previously spread out

-----

There is more to be done, but I wanted to get some feedback before continuing - is this ok so far?

### CODE REVIEW

File: src/api/app/assets/stylesheets/webui/application/bento/base.scss [hunk 1]
```diff
@@ -96,38 +91,112 @@ blockquote{border-left:5px solid #690;color:#666;font-style:italic;}
...
+    border-color:transparent #333 #111;
+    opacity:0.95;
+    box-shadow:0 1px 3px #999;
+
+    input{width:208px;font-size:1.5em;padding:2px 3px;border:3px solid #222;border-radius:5px;}
+    input:focus{color:#39c;border-color:#39c;box-shadow:0 1px 3px #222;}
+
+    .slim-footer{text-align:center;margin-bottom:0;}
+    .slim-footer a{text-decoration:none;color:#eee;font-weight:bold;}
+    .slim-footer a:hover{text-decoration:underline;}
```
Comment (mdeniz) [inline src/api/app/assets/stylesheets/webui/application/bento/base.scss]: Can be `.slim-footer` done in the cascading way?
```css
.slim-footer {
  text-align: center;
  margin-bottom: 0;
  a {
     text-decoration: none;
     color: #eee;
     font-weight: bold;
     &:hover {
         text-decoration: underline;
      }
   }
}
```

Comment (avindra) [inline src/api/app/assets/stylesheets/webui/application/bento/base.scss]: Sure, I will update the PR to use this style for more of the rules. Wanted to get feedback first :+1:

Comment (mdeniz) [inline src/api/app/assets/stylesheets/webui/application/bento/base.scss]: Ok, then keep going! :smile:

**Comment by mdeniz** [comment]
@avindra I will merge this as it is... create a new PR for more changes, many thanks!

**Pull request merged**